doc/ledger3.texi

```diff
@@ -2919,7 +2919,7 @@ after the amount, or on its own line preceded by whitespace:
 @c > Is there a way to produce a register report that lists all the transaction
 @c > that contain a certain tag, and sort them based on the value of the tag?
 @c ledger reg --sort "tag('foo')" %foo
-@c ledger reg --group-by "tag('Employer)" Remboursement:Employer and tag Employer
+@c ledger reg --group-by "tag('Employer)" Reimbursement:Employer and tag Employer
 @c > Is it possible to get subtotals for each tag value?
```

Comment:
"Employer", in French, is a verb that would mean something like "to hire sb"
"Employeur" in French, is the noun for English "Employer"
I admit that here, as commit author, I have meant Employeur.
The commit from jostber is acceptable.
I just like the fact, that some examples in the ledger documentation stays in the original language, as there are others example like that, this make ledger more universal from my point of view ! :-)